core/src/main/java/de/danoeh/antennapod/core/service/GpodnetSyncService.java

```diff
@@ -53,14 +56,19 @@
     private static final String ACTION_SYNC_SUBSCRIPTIONS = "de.danoeh.antennapod.intent.action.sync_subscriptions";
     private static final String ACTION_SYNC_ACTIONS = "de.danoeh.antennapod.intent.action.sync_ACTIONS";
 
-    private GpodnetService service;
+    private static final AtomicInteger syncActionCount = new AtomicInteger(0);
 
+    private GpodnetService service;
     private boolean syncSubscriptions = false;
     private boolean syncActions = false;
```

Comment:
Not sure if a JobService is executed in a new object each time, but either `syncActionCount` should be non-static or those should be static.